### PR TITLE
tour-guide CLI.

### PR DESCRIPTION
**Pull request by dmabtrg**
Mostly a skeleton with placeholders for the future commands that must be supported.

### CODE REVIEW

**Comment by FiwareULPGC** [comment]
LGTM

**Comment by dmabtrg** [comment]
I've updated the PR with more commands for the CLI script.

File: scripts/check [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+Command options:
+
+  -h  --help                 Show this help.
+
+EOF
+   exit 0
+}
+
+function module_options () {
+
```
Comment (FiwareULPGC) [inline scripts/check]: Unneeded line.

File: scripts/check [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    shift
+
+    if [ $# -gt 0 ]; then
+        echo "Unknown parameters: $@"
+        module_help
+    fi
+}
+
+function check_cmd () {
+
```
Comment (FiwareULPGC) [inline scripts/check]: Unneeded line.

File: scripts/sensors_simulate-data [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+    local old_value=$( curl --silent \
+                            --show-error \
+                            --header "Fiware-Service: ${fiware_service}" \
+                            --header "Accept: text/plain" \
+                            "http://localhost:1026/v2/entities/${restaurant_id}/attrs/${type}:${room}/value" )
+    echo ${old_value}
+}
+
+function generate_sensor_value () {
+
```
Comment (FiwareULPGC) [inline scripts/sensors_simulate-data]: Unneeded line.

File: scripts/sensors_simulate-data [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+    fi
+
+    if [ -z "${sensors_delay}" ] ; then
+        echo "Missing parameter '--delay'." && missing_parameters=1
+    else
+        [[ ! ${sensors_delay} =~ ^\+?[0-9]+$ ]] && echo "Delay value must be a positive integer." && missing_parameters=1
+        [ ${sensors_delay} -lt 1 ] && echo "Delay value must be greater than 0." && missing_parameters=1
+    fi
+
+
```
Comment (FiwareULPGC) [inline scripts/sensors_simulate-data]: Unneeded line.

File: scripts/sensors_simulate-data [hunk 5]
```diff
@@ -0,0 +1,197 @@
...
+    [ ${missing_parameters} -ne 0 ] && submodule_help
+
```
Comment (FiwareULPGC) [inline scripts/sensors_simulate-data]: Unneeded line.

File: scripts/start [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+Command options:
+
+  -h  --help                 Show this help.
+
+EOF
+   exit 0
+}
+
+function module_options () {
+
```
Comment (FiwareULPGC) [inline scripts/start]: Unneeded line.

File: scripts/stop [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+  -h  --help                 Show this help.
+  -r  --remove               Remove docker containers after stopping them.
+                             Default is NOT to remove the containers.
+
+EOF
+   exit 0
+}
+
+function module_options () {
+
```
Comment (FiwareULPGC) [inline scripts/stop]: Unneeded line.

**Comment by FiwareULPGC** [comment]
It would be good to have a README file with a short explanation about the scripts.

**Comment by FiwareULPGC** [comment]
We had some problems trying to create sensors. We followed the instructions provided by  the `tour-guide sensors create --help`  command. 

The error can be seen in the attached picture.
error-create

**Comment by FiwareULPGC** [comment]
We also had problems trying to configure keyrock and oauth.
oauth-keyrock-errors

**Comment by FiwareULPGC** [comment]
We think that the purpose of this PR is to facilitate the use and configuration of the tourguide application. If this is the case, we miss an option to set the hosts file.

**Comment by dmabtrg** [comment]
This PR will be splitted into smaller ones.

**Pull request closed without merging**